gitlab/v4/objects/files.py

```diff
@@ -31,7 +36,11 @@ def decode(self) -> bytes:
         """
         return base64.b64decode(self.content)
 
-    def save(self, branch, commit_message, **kwargs):
+    # NOTE(jlvillal): Signature doesn't match SaveMixin.save() so ignore
+    # type error
+    def save(  # type: ignore
```

Comment:
For future, I wonder if `@typing.overload` with all the possible signatures might help here. Disclaimer again this is just from reading I haven't tested that stuff 😀